package/kedro_viz/server.py

```diff
@@ -118,12 +122,20 @@ def run_server(
     parser = argparse.ArgumentParser(description="Launch a development viz server")
     parser.add_argument("project_path", help="Path to a Kedro project")
     parser.add_argument(
-        "--host", help="The host of the development server", default=_DEFAULT_HOST
+        "--host", help="The host of the development server", default="localhost"
     )
     parser.add_argument(
         "--port", help="The port of the development server", default=4142
     )
```

Comment:
does `_DEFAULT_HOST` serve any use anymore? Seems like it's just the default argument to `run_server` -- when is `run_server` used outside of `__main__`? Someone importing `kedro_viz` and running it manually?  Should we replace the 4142 below with `_DEFAULT_PORT` and replace `_DEFAULT_HOST="localhost"`?

Reply:
Agree with @mzjp2 we could probably remove the constants from the top of the file now.

Reply:
Well... actually, there are 2 ways to launch a server here: the `localhost:4142` are for us viz devs to launch viz in development mode. The `DEFAULT_HOST:DEFAULT_PORT` are for end-users. I'm changing this to

```
_DEFAULT_HOST="localhost"
_DEFAULT_PORT=4141
_DEV_PORT=4142
```

Reply:
Ah, sorry I saw 4142 and 4141 and thought they were the same 🤦 -- your new commit tidies thigns up nicely though!!